### PR TITLE
Helm chart and file Setup

### PR DESCRIPTION
**Pull request by nghe0001**
-Helmfiles and Helm chart imported to repo and setup

### CODE REVIEW

File: helm/charts/alphasite/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: Aplha Site
```
Comment (will0684) [inline helm/charts/alphasite/Chart.yaml:3]: typo ;)

**Review by will0684 — CHANGES_REQUESTED** [review]

**Review by Moro-Code — CHANGES_REQUESTED** [review]
did you test this against the dev cluster ?

File: helm/charts/alphasite/Chart.yaml (hunk 1 above)
Comment (nghe0001) [reply helm/charts/alphasite/Chart.yaml]: fixed =)

**Comment by nghe0001** [comment]
> did you test this against the dev cluster ?

did not have a img yesterday to test. Just got it tested with Adam this morning and is working fine now

**Review by Moro-Code — APPROVED** [review]
Good to merge

**Review by will0684 — APPROVED** [review]

**Pull request merged**